### PR TITLE
add Bulma and write test for search reducer

### PR DESCRIPTION
**Pull request by auredufour**
Added Bulma as the Css framework and font awesome.
http://bulma.io/documentation/overview/start/
and create the search reducer test

### CODE REVIEW

**Comment by srhboo** [comment]
👍

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,9 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+Copy
```
Comment (cgolinski) [inline public/index.html]: Is 'copy' here by mistake?

**Comment by cgolinski** [comment]
Perhaps change the name of this PR, as it doesn't include a test for search reducer.

File: public/index.html (hunk 1 above)
Comment (auredufour) [inline public/index.html]: yes! I will remove it

**Pull request merged**